xscen/ensembles.py

```diff
@@ -162,6 +163,14 @@ def generate_weights(
         'institution': Weights using the method '1 institution - 1 Vote'
     experiment_weights : bool
```

Comment:
Do I change experiment_weights to balance_experiments or experiments_independance?

Reply:
Probably `balance_experiments`, yeah. To do this properly (although I doubt that anyone really used this, since it's pretty new), I would still support `experiment_weights`, but add a `FutureWarning`.

Something like:
```
if experiment_weights is True:
    warnings.warn(
        "`experiment_weights` has been renamed and will be removed in a future release. Use `balance_experiments` instead.",
        category=FutureWarning,
    )
    balance_experiments = True
```